Directory.Build.props

```diff
@@ -12,7 +12,6 @@
     <EnableNETAnalyzers>true</EnableNETAnalyzers>
     <AnalysisMode>Recommended</AnalysisMode>
     <PlatformTarget>$(Platform)</PlatformTarget>
-    <Platforms>$(Platform)</Platforms>
```

Comment:
why remove it?

Reply:
Because it's up there in the file already. This is a re-declaration.